distributions/aws/test/e2e/fixtures/storage_fsx_dependencies.py

```diff
@@ -0,0 +1,249 @@
+import time
+import pytest
+import subprocess
+import boto3
+
+from e2e.utils.config import metadata
+from e2e.utils.cognito_bootstrap.common import load_cfg, write_cfg
+from e2e.conftest import region
+from e2e.fixtures.cluster import cluster
+from e2e.utils.utils import rand_name
+from e2e.utils.config import configure_resource_fixture
+from e2e.fixtures.cluster import associate_iam_oidc_provider, create_iam_service_account
+from e2e.utils.utils import (
+    rand_name,
+    wait_for,
+    get_iam_client,
+    get_eks_client,
+    get_ec2_client,
+    get_fsx_client,
+    curl_file_to_path,
+    get_aws_account_id,
+    kubectl_apply,
+    kubectl_delete,
+    kubectl_apply_kustomize,
+    kubectl_delete_kustomize,
+)
+
+DEFAULT_NAMESPACE = "kube-system"
+
+
+def wait_on_fsx_status(desired_status, fsx_client, file_system_id):
+
+    def callback():
+        response = fsx_client.describe_file_systems(
+            FileSystemIds=[
+                file_system_id
+            ]
+        )
+        filesystem_status = response["FileSystems"][0]["Lifecycle"]
+        print(f"{file_system_id} {filesystem_status} .... waiting")
+        assert filesystem_status == desired_status
+        
+    wait_for(callback, 600)
+
+def get_fsx_dns_name(fsx_client, file_system_id):
+    response = fsx_client.describe_file_systems(
+            FileSystemIds=[
+                file_system_id
+            ]
+        )
+    return response["FileSystems"][0]["DNSName"]
+
+def get_fsx_mount_name(fsx_client, file_system_id):
+    response = fsx_client.describe_file_systems(
+            FileSystemIds=[
+                file_system_id
+            ]
+        )
+    return response["FileSystems"][0]["LustreConfiguration"]["MountName"]
+
+@pytest.fixture(scope="class")
+def install_fsx_csi_driver(metadata, region, request, cluster):
+    fsx_driver = {}
+    FSx_DRIVER_VERSION = "v0.7.1"
+    FSx_CSI_DRIVER = f"github.com/kubernetes-sigs/aws-fsx-csi-driver/deploy/kubernetes/overlays/stable/?ref=tags/{FSx_DRIVER_VERSION}"
+
+    def on_create():
+        kubectl_apply_kustomize(FSx_CSI_DRIVER)
+        # cmd = f"kubectl apply -k {FSx_CSI_DRIVER}".split()
```

Comment:
similar question why is this commented out?

Reply:
Needs cleanup in both EFS and FSx - was testing something out.